chrome/content/zotero/xpcom/rdf/n3parser.js

```diff
@@ -31,7 +31,7 @@ $rdf.N3Parser = function () {
 
   var pyjslib_slice = function (str, i, j) {
       if(typeof str.slice == 'undefined')
-        throw '@@ mising.js: No .slice function for ' + str + ' of type ' + (typeof str)
+        throw '@@ missing.js: No .slice function for ' + str + ' of type ' + (typeof str)
```

Comment:
Files in the `rdf` directory aren't mostly third-party files and shouldn't be modified.